src/components/command-bar/commands/chat/index.tsx

```diff
@@ -17,7 +17,7 @@ const Icon = () => {
 		<IconArrowLeft24
```

Comment:
Note this whole component should be a button and not an image with interactivity added. That would need to be a bigger refactor so pushing this update for now and separating that out. Issue is noted [in this doc ](https://docs.google.com/document/d/17VH69KP2TFNDkC7MEdP8-wUnec4UtcXfMUlCPX7eE6Y/edit#heading=h.d71g029sucyd)